docs/tutorials/01_neural_networks.ipynb

```diff
@@ -7,90 +7,132 @@
    "source": [
     "# Quantum Neural Networks\n",
     "\n",
-    "This notebook demonstrates the different generic quantum neural network (QNN) implementations provided in Qiskit Machine Learning.\n",
-    "The networks are meant as application-agnostic computational units that can be used for many different use cases. \n",
-    "Depending on the application, a particular type of network might be more or less suitable and might require to be set up in a particular way.\n",
-    "The following different available neural networks will now be discussed in more detail:\n",
+    "## Overview\n",
+    "This notebook demonstrates different quantum neural network (QNN) implementations provided in `qiskit-machine-learning`, and how they can be integrated into basic quantum machine learning (QML) workflows.\n",
     "\n",
-    "1. `NeuralNetwork`: The interface for neural networks. This is an abstract class.\n",
-    "2. `EstimatorQNN`: A network based on the evaluation of quantum mechanical observables.\n",
-    "3. `SamplerQNN`: A network based on the samples resulting from measuring a quantum circuit.\n",
-    "\n",
-    "Each implementation, `EstimatorQNN` and `SamplerQNN`, takes in an optional instance of the corresponding Qiskit primitive, namely `BaseEstimator` and `BaseSampler`. The latter two define two interfaces of the primitives. Qiskit provides the reference implementation as well as a backend-based implementation of the primitives. The primitives is a frontend to either a simulator or a real quantum hardware. By default, if no instance is passed to a network, an instance of the Qiskit reference primitive is created automatically by the network. For more information about primitives please refer to [Qiskit primitives](https://qiskit.org/documentation/apidoc/primitives.html)."
+    "The tutorial is structured as follows:\n",
+    "1. [Introduction](#Introduction)\n",
+    "2. [How to Instantiate QNNs](#Instantiate)\n",
+    "3. [How to Run a Forward Pass](#Forward)\n",
+    "4. [How to Run a Backward Pass](#Backward)\n",
+    "5. [Advanced Functionality](#Advanced)\n",
+    "6. [Conclusion](#Conclusion)"
```

Comment:
Please correct the structure, in the generated documentation it looks like here and the links don't work.
<img width="891" alt="image" src="https://user-images.githubusercontent.com/62334182/213701820-7f453cb7-6ac2-4462-a8b6-51a38f5e8ff7.png">

Reply:
All level two headers are broken, actually.

Reply:
I have fixed both the index and the headers.